### PR TITLE
Table and management service for balance notification settings

### PR DESCRIPTION
**Pull request by Raibo**
https://github.com/happy-travel/agent-app-project/issues/567

### CODE REVIEW

File: HappyTravel.Edo.Data/Agents/BalanceNotificationSetting.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace HappyTravel.Edo.Data.Agents
+{
+    public class BalanceNotificationSetting
+    {
+        public int Id { get; set; }
+        public int AccountId { get; set; }
```
Comment (mingazhev) [inline HappyTravel.Edo.Data/Agents/BalanceNotificationSetting.cs:6]: AgencyAccountId maybe? To be more clear

File: Api/Models/Agencies/BalanceNotificationSettingInfo.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace HappyTravel.Edo.Api.Models.Agencies
+{
+    public readonly struct BalanceNotificationSettingInfo
+    {
+        public int AccountId { get; init; }
```
Comment (mingazhev) [inline Api/Models/Agencies/BalanceNotificationSettingInfo.cs:5]: Is accountId needed in this model?

**Comment by mingazhev** [comment]
Added some comments

File: Api/Models/Agencies/BalanceNotificationSettingInfo.cs (hunk 2 above)
Comment (Raibo) [reply Api/Models/Agencies/BalanceNotificationSettingInfo.cs:5]: Technically no, but I just wanted the model to be self-sufficient and unambiguous, for this I added AccountId.

**Review by mingazhev — APPROVED** [review]

**Pull request merged**